packages/contracts/abis/WETH.json

```diff
@@ -0,0 +1,153 @@
+[
```

Comment:
nit: Write in typescript to ensure type guarding?

Reply:
We can import typing from the `web3-utils` lib.

```
import type { AbiItem } from 'web3-utils'
```